data/transactions/logic/doc.go

```diff
@@ -242,10 +242,10 @@ var opcodeImmediateNotes = map[string]string{
 	"gtxnas":  "{uint8 transaction group index} {uint8 transaction field index}",
 	"gtxnsas": "{uint8 transaction field index}",
 
-	"bnz":     "{int16 branch offset, big-endian}",
-	"bz":      "{int16 branch offset, big-endian}",
-	"b":       "{int16 branch offset, big-endian}",
-	"callsub": "{int16 branch offset, big-endian}",
+	"bnz":     "{int16 branch offset}",
```

Comment:
The `big-endian` note seems to imply there might be a little endian encoding somewhere but afaik there isnt?

If you'd like to keep this one, I'll add it to the frame_* ops that have a signed int type in their note as well

Reply:
It's only here because this is a u16. single byte types (including the frame_* stuff) can't be big or little endian.

Reply:
ah right.

I'll restore `big-endian`